lib/rubocop/graphql/argument/kwargs.rb

```diff
@@ -20,7 +20,7 @@ class Kwargs
         PATTERN
 
         def initialize(argument_node)
-          @nodes = argument_kwargs(argument_node)
+          @nodes = argument_kwargs(argument_node) || []
```

Comment:
I kept this here because I realized that the node matchers are still able to match legacy definitions. I opted not to add a legacy based spec, so that people aren't encouraged to add specs to support legacy versions. The motivation for keeping this here was so that people could actually use the gem, otherwise they will need to disable this cop and it may be a higher barrier to entry. 

What do you think? I think we should keep this change, as it will be easier for people to install the gem (and still matches the legacy nodes). Users will also get the warning to migrate to the class-based definition. I also think this would help with the graphQL-ruby philosophy of migrating to class-based API in a step-based fashion ([source](https://graphql-ruby.org/schema/class_based_api.html#compatibility--migration-overview)).

Reply:
I wonder how many gems we'd have to fix to make it work with block–based API, but yeah, let's keep it 🙂

Reply:
@DmitryTsepelev it works quite well! The code I am working with is currently legacy-based and I am migrating a lot of it over to class-based. However I want to set it up so that people don't accidentally add legacy-based block syntax as well 🙂